core/templates/pages/release-coordinator-page/release-coordinator-page.component.html

```diff
@@ -73,9 +76,159 @@ <h3>Enable Promo Bar</h3>
+                    <mat-chip *ngFor="let user of userGroup.userGroupUserUsernames"
+                              [selectable]="false"
+                              [removable]="false">
+                      <ng-container>
+                        <span tabindex="0" [attr.aria-label]="user+', selected'">{{ user }}</span>
+                      </ng-container>
+                    </mat-chip>
+                  </mat-chip-list>
+                </div>
+                <div *ngIf="userGroupInEditMode">
+                  <mat-chip-list #chipList aria-label="Users Associated With The User Group" tabindex="0">
+                    <mat-chip *ngFor="let user of userGroup.userGroupUserUsernames"
+                              [selectable]="true"
+                              [removable]="true"
+                              (removed)="removeUserFromUserGroup(userGroup, user)">
+                      <ng-container>
+                        <span tabindex="0" [attr.aria-label]="user+', selected'">{{ user }}</span>
+                      </ng-container>
+                      <button class="oppia-cancel-button" role="button" tabindex="0" matChipRemove aria-label=". Remove">
+                        <mat-icon class="oppia-cancel-button-text">cancel</mat-icon>
+                      </button>
+                    </mat-chip>
+                    <input placeholder="Add Username"
+                           #userInputToAddUserToGroup
+                           [matChipInputFor]="chipList"
+                           [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
+                           (matChipInputTokenEnd)="addUserToUserGroup($event, userGroup)"
+                           (input)="onUserGroupUserInputChange()">
+                  </mat-chip-list>
+                  <p *ngIf="userInUserGroupValidationError.length > 0"
+                     class="oppia-warning-text">
+                    {{ userInUserGroupValidationError }}
+                  </p>
+                </div>
+              </div>
+            </div>
+            <hr *ngIf="userGroupInEditMode">
+            <p *ngIf="userGroupSaveError.length > 0"
+               class="oppia-warning-text">
+              {{ userGroupSaveError }}
+            </p>
+            <div *ngIf="userGroupInEditMode" class="row justify-content-end">
+              <button mat-raised-button
+                      type="button"
+                      (click)="resetUserGroup(userGroup)">
+                Cancel
+              </button>
+              <button mat-raised-button
+                      color="primary"
+                      type="submit"
+                      (click)="updateUserGroup(userGroup)">
+                Save
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="oppia-add-user-group">
+        <input placeholder="New UserGroup Name"
```

Comment:
New user group name

(For user-facing text, in general, "user group" is 2 words.)

Reply:
Updated